mantle/kola/tests/misc/network.go

```diff
@@ -62,14 +62,16 @@ func init() {
 	// with a slight change, where the script originally run by MCO is run from
 	// ignition: https://github.com/RHsyseng/rhcos-slb/blob/161a421f8fdcdb4b08fb6366daa8fe1b75cbe095/init-interfaces.sh.
 	register.RegisterTest(&register.Test{
-		Run:         InitInterfacesTest,
-		ClusterSize: 0,
-		Name:        "rhcos.network.init-interfaces-test",
-		Description: "Verify init-interfaces script works in both fresh setup and reboot.",
-		Timeout:     40 * time.Minute,
-		Distros:     []string{"rhcos"},
-		Platforms:   []string{"qemu"},
-		Tags:        []string{"openshift"},
+		Run:            InitInterfacesTest,
+		ClusterSize:    1,
+		Name:           "rhcos.network.init-interfaces-test",
+		Description:    "Verify init-interfaces script works in both fresh setup and reboot.",
+		Timeout:        40 * time.Minute,
+		Distros:        []string{"rhcos"},
+		Platforms:      []string{"qemu"},
+		Tags:           []string{"openshift"},
+		AdditionalNics: 2,
```

Comment:
so previously `RegisterTest()` didn't know about `AdditionalNics` and the test had to set our own `QemuMachineOptions` to get that functionality?

Reply:
I *think* that's what happened but I didn't verify it. It's entirely possible that the test was written after support was added, but no one involved in the writing/review of the test realized it could've been simplified.